charts/openmetadata/templates/ingress.yaml

```diff
@@ -17,9 +17,9 @@ kind: Ingress
 metadata:
   name: {{ $fullName }}
   labels:
-    {{- include "OpenMetadata.labels" . | nindent 4 }}
-  {{- with .Values.ingress.annotations }}
+    {{- include "OpenMetadata.labels" . | nindent 4 }} 
   annotations:
+    {{- with .Values.ingress.annotations }}
```

Comment:
All the Kubernetes Objects should refer to this by calling
`{{- include "OpenMetadata.annotations" . | nindent 4 }}`.

Refactor to _helper.tpl as mentioned above in review.

Reply:
will keep this condition here as this is for Github Actions datree - 
```
{{- if semverCompare "=<1.13-0" $.Capabilities.KubeVersion.GitVersion }}
datree.skip/K8S_DEPRECATED_APIVERSION_1.16: "Ignore that deprecation in old kubernetes instances"
{{- end }}
```

Reply:
added {{- include "OpenMetadata.annotations" . | nindent 4 }}